modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -348,7 +348,9 @@ export class Tileset3D {
     if (isDataUrl) {
       return tilePath;
     }
-    return `${tilePath}${tilePath.includes('?') ? '&' : '?'}${this.queryParams}`;
+    return `${tilePath}${tilePath.includes('?') ? '&' : this.queryParams.length ? '?' : ''}${
```

Comment:
I think this code is too dense, could you split across multiple lines? You're handling the `queryParams = ''` case, but it still isn't perfect as `&` is needlessly appended if the tilePath contains `?`

Reply:
Thanks, agree. Please review updated version.